commit-message-guidelines.md

```diff
@@ -0,0 +1,32 @@
+# Commit Message Guidelines
+
+[Atomic commits](https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/) are preferred.
+
+## Subject/Title
+
+* Separate the title/subject of the commit message from the body of the commit message with a blank line.
+* Try to keep the title/subject descriptive and concise.
+* Capitalize the title/subject line, use the imperative mood, and don't end the line with a period.
+
+## Body
+
+* Explain what the commit does.
+* Explain why the commit does what it does.
+* Add any additional details that could be useful.
```

Comment:
Should a commit related to an issue mention the issue in the commit message (if so, in title or body) or just the PR?